.travis.yml

```diff
@@ -3,27 +3,27 @@
 
 language: python
 python:
-  - 3.5
-  - 3.4
-  - 3.3
-  - 2.7
-  - 2.6
+    - 3.6
 
 # command to install dependencies, e.g. pip install -r requirements.txt --use-mirrors
-install: pip install -U tox-travis
+install:
+    pip install -U tox-travis
+
 
 # command to run tests, e.g. python setup.py test
-script: tox
+script:
+    touch /home/travis/.xenon_config
+    tox
 
 # After you create the Github repo and add it to Travis, run the
 # travis_pypi_setup.py script to finish PyPI deployment setup
-deploy:
-  provider: pypi
-  distributions: sdist bdist_wheel
-  user: XeBoris
-  password:
-    secure: PLEASE_REPLACE_ME
-  on:
-    tags: true
-    repo: XeBoris/admix
-    python: 2.7
+#deploy:
```

Comment:
Can we just delete instead of commenting?

Reply:
agree, done in e77dd0e